docs/changelog/89212.yaml

```diff
@@ -3,3 +3,27 @@ summary: Add an unattended mode setting to transform
 area: Transform
 type: enhancement
 issues: []
+highlight:
+  title: Unattended mode for transforms
+  body: |-
+    The new `unattended` setting for transforms introduces unattended mode. It 
+    enables the transform to retry indefinitely, even in the face of errors that 
+    require direct intervention to resolve. For example, a high disk watermark 
+    may prevent all ingest activities, including the indexing operation of a 
+    transform; however once the original issue is resolved by the system 
+    administrator, the transform automatically continue to operate from where it 
```

Comment:
There's something missing.  Could be "the transform will automatically" or "automatically continues" - whatever matches the house style best.